tiatoolbox/utils/metrics.py

```diff
@@ -0,0 +1,82 @@
+# ***** BEGIN GPL LICENSE BLOCK *****
+#
```

Comment:
Do we instead want to have tiatoolbox.metrics ? I am okay for this PR but we may want to consider a refactor. How do they do it in SKlearn ?

Reply:
Kept under a folder but being exposed under `sklearn.metrics` namespace. I think we should just leave them here for now.

https://github.com/scikit-learn/scikit-learn/tree/0d378913be6d7e485b792ea36e9268be31ed52d0/sklearn/metrics